changelog/dip1000_deprecation_warnings.dd

```diff
@@ -0,0 +1,6 @@
+Print warning messages unless -revert=dip1000 is used
+
+As an incentive to transition to -preview=dip1000 by default, now the
+compiler prints what would have been error messages if the flag had
+been on as deprecation messages.  This new behaviour can be disabled
```

Comment:
Only in explicit `@safe` functions, function with inference may still silently be `@safe` while having `scope` violations.